deploy-agent/deployd/client/client.py

```diff
@@ -154,6 +155,9 @@ def _read_host_info(self):
             if not self._stage_type and not self._stage_type_fetched:
                 keys_to_fetch.add(stage_type_key)
 
+            if not self._account_id:
+                keys_to_fetch.add("ec2_metadata.identity-credentials.ec2.info")
```

Comment:
add a helper function in config.py with default value. such as  https://sourcegraph.pinadmin.com/github.com/pinterest/teletraan/-/blob/deploy-agent/deployd/common/config.py?L259

Reply:
right, make sense